dart/lib/src/sentry.dart

```diff
@@ -1,15 +1,22 @@
 import 'dart:async';
 
+import 'default_integrations.dart';
 import 'hub.dart';
 import 'hub_adapter.dart';
 import 'noop_hub.dart';
+import 'noop_isolate_error_integration.dart'
+    if (dart.library.io) 'isolate_error_integration.dart';
```

Comment:
does it need to be a NoOp import? did we have a bug on the previous code which added or not the integration? just a question though

Reply:
without this conditional import you can't compile for the web

Reply:
we didn't have this `noop` before and it was compiling the Flutter web project, wondering why it didn't break before

Reply:
Unfortunately I didn't ran the example with Flutter web for a while. There was a Flutter web compilation error, this PR fixes it.

Reply:
I see, but CI should have broken this https://github.com/getsentry/sentry-dart/blob/main/.github/workflows/flutter.yml#L62
wondering if this works at all (the CI)

Reply:
i created a task for it (CI)